Landing page wip

reedwane opened the pull request.

TopRateANZ commented:
There is no Screenshort on this PR

TopRateANZ commented on Dockerfile:
```diff
@@ -0,0 +1,59 @@
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  elif [ -f package-lock.json ]; then npm ci; \
+  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i --frozen-lockfile; \
```
Clean Up

TopRateANZ commented:
Opacity is overshadowing content . The hero text should be on 2 lines not 3 and the size should be reduced. You can look at 
https://topratetransfer.com/#/  as a Guide

TopRateANZ commented:
Size or Zoom Issues on Landing Page

TopRateANZ commented:
FORGET PASSWORD Page Should have a Country Dropdown on the Left where country code is selected . If it's possible for the dropdown

TopRateANZ commented:
Clicking on the Logo Should Redirect to the Landing Page

TopRateANZ commented:
Drop dpown for Sending Country Should look like https://wise.com/  atleast 60-70% of the width

TopRateANZ commented:
All Dropdowns should have padding on the right

reedwane commented:
<img width="539" alt="image" src="https://github.com/TopRateTransfer-Pty-Ltd/TopRateMax.LandingAuth.Web.Repo/assets/68650188/29646b72-837b-44dc-870b-7e4b3525f1b6">

I took care of the opacity as requested, and adjusted the currency dropdown

reedwane commented:
<img width="957" alt="image" src="https://github.com/TopRateTransfer-Pty-Ltd/TopRateMax.LandingAuth.Web.Repo/assets/68650188/09fc07d5-c8e9-4b5f-ab41-3d44b5341c1a">

I have adjusted the hero as requested. Also adjusted the opacity of the overlay on the video.
